app/experimenter/experiments/api/v5/query.py

```diff
@@ -74,6 +75,9 @@ def resolve_hypothesis_default(root, info):
     def resolve_max_primary_probe_sets(root, info):
         return NimbusExperiment.MAX_PRIMARY_PROBE_SETS
 
+    def resolve_documentation_link(root, info):
+        return root._text_choices_to_label_value_list(NimbusExperiment.DocumentationLink)
```

Comment:
This needs to get worked into `test_nimbus_config` in `app/experimenter/experiments/tests/api/v5/test_query.py` to pad out coverage - something like this:

```diff
diff --git a/app/experimenter/experiments/tests/api/v5/test_query.py b/app/experimenter/experiments/tests/api/v5/test_query.py
index 4e65eef0..33b00993 100644
--- a/app/experimenter/experiments/tests/api/v5/test_query.py
+++ b/app/experimenter/experiments/tests/api/v5/test_query.py
@@ -356,6 +356,10 @@ class TestNimbusQuery(GraphQLTestCase):
                         label
                         value
                     }
+                    documentationLink {
+                        label
+                        value
+                    }
                     hypothesisDefault
                     maxPrimaryProbeSets
                 }
@@ -377,6 +381,7 @@ class TestNimbusQuery(GraphQLTestCase):
         assertChoices(config["channel"], NimbusExperiment.Channel)
         assertChoices(config["firefoxMinVersion"], NimbusExperiment.Version)
         assertChoices(config["targetingConfigSlug"], NimbusExperiment.TargetingConfig)
+        assertChoices(config["documentationLink"], NimbusExperiment.DocumentationLink)
         self.assertEqual(len(config["featureConfig"]), 10)
         self.assertEqual(len(config["probeSets"]), 10)
         app_channels = config["applicationChannels"]
```